proms-resources/frontend/src/main/frontend/src/MockPatientIdentification.jsx

```diff
@@ -280,7 +328,33 @@ function MockPatientIdentification(props) {
     return null;
   }
 
-  return (
+  return (<>
+   <ResponsiveDialog title="Terms of Use" open={showTou} width="lg" >
+     <DialogContent dividers>
+       <FormattedText variant="caption">{touText}</FormattedText>
```

Comment:
`caption` is too small, let's use the default instead.
```suggestion
       <FormattedText>{tou.text}</FormattedText>
```